docs/quickstart/php.md

```diff
@@ -71,6 +71,10 @@ $ cd Protobuf-PHP
 $ rake pear:package version=1.0
 $ [sudo] pear install Protobuf-1.0.tgz
 ```
+> maybe you should install 'rake' or 'ronn' if they not installed before:
```

Comment:
Please change this line to 

"If you did not have 'rake' or 'ronn' installed, please install them using this command:"